src/Widgets/ExportWidget.vala

```diff
@@ -108,33 +105,32 @@ public class Akira.Widgets.ExportWidget : Gtk.Grid {
         yield get_image_buffer_size ();
 
         double bytes = (double) imagedata.length;
-        double full_bytes = imagedata.length > MB
-            ? bytes / MB : bytes / KB;
+        double full_bytes = imagedata.length > MB ? bytes / MB : bytes / KB;
         var size = imagedata.length > MB
             ? ("%0.1fMB").printf (full_bytes)
             : ("%0.1fKB").printf (full_bytes);
```

Comment:
any reason not to use [GLib.format_size](https://valadoc.org/glib-2.0/GLib.format_size.html)?

Reply:
Uh, thanks, I wasn't aware of that T_T